### PR TITLE
e2e test for dockerfile and pipeline

### PR DESCRIPTION
**Pull request by coolteddy**
e2e test for uat and dockerfile update for geckodriver and firefox installation.

### CODE REVIEW

File: pipelines/csw-test-pipeline.yml [hunk 1]
```diff
@@ -40,7 +100,7 @@ resources:
     url: ((slack-webhook-cyber-test))
 
 jobs:
-- name: job-csw-test-build
+- name: build
```
Comment (elisha512) [inline pipelines/csw-test-pipeline.yml:103]: should this be called uat-build for consistency or is build simply the same for all env?

File: pipelines/csw-test-pipeline.yml [hunk 2]
```diff
@@ -161,23 +256,54 @@ jobs:
...
+        - -c
+        - |
+
+          echo "deploying to ${ENVIRONMENT}..."
 
     on_success:
       <<: *success_notify
 
-- name: job-csw-test-cleanup
+- name: cleanup
```
Comment (elisha512) [inline pipelines/csw-test-pipeline.yml:306]: again should this be uat-cleanup?

**Review by elisha512 — COMMENTED** [review]
Left two comments in there, otherwise looks good

File: pipelines/csw-test-pipeline.yml (hunk 1 above)
Comment (coolteddy) [reply pipelines/csw-test-pipeline.yml:103]: this job is general, we could build any environment with this. But we are not going to use that for now.

File: pipelines/csw-test-pipeline.yml (hunk 2 above)
Comment (coolteddy) [reply pipelines/csw-test-pipeline.yml:306]: hope we don't need to do that :) see above comment

**Review by elisha512 — APPROVED** [review]

**Pull request merged**